Tasks/PowerShellV2/powershell.ps1

```diff
@@ -42,6 +42,9 @@ try {
     Write-Host (Get-VstsLocString -Key 'GeneratingScript')
     $contents = @()
     $contents += "`$ErrorActionPreference = '$input_errorActionPreference'"
+    # Change default error view to normal view. We need this for error handling since we pipe stdout and stderr to the same stream
```

Comment:
Nit: Add this after sentence 1:
(required because PowerShell Core changed the default to ConciseView)